service/src/main/resources/changelog/changesets/20230125_assemble_refbased_io_updates.yaml

```diff
@@ -0,0 +1,1417 @@
+databaseChangeLog:
+  - changeSet:
+      id: "update assemble_refbased inputs/outputs to match the target workspace for public preview"
+      author: mspector
+      changes:
+        - update:
+            tableName: run_set
+            where: run_set_id='10000000-0000-0000-0000-000000000005'
+            columns:
+              - column:
+                  name: input_definition
+                  value: |
+                    [
+                      {
+                        "input_name": "assemble_refbased.reads_unmapped_bams",
+                        "input_type": {
+                          "type": "array",
+                          "array_type": {
+                            "type": "primitive",
+                            "primitive_type": "File"
+                          },
+                          "non_empty": true
+                        },
+                        "source": {
+                          "type": "record_lookup",
+                          "record_attribute": "srr_ubam"
+                        }
+                      },
+                      {
+                        "input_name": "assemble_refbased.reference_fasta",
+                        "input_type": {
+                          "type": "primitive",
+                          "primitive_type": "File"
+                        },
+                        "source": {
+                          "type": "record_lookup",
+                          "record_attribute": "workspace.ncov_reference_genome_fasta"
+                        }
+                      },
+                      {
+                        "input_name": "assemble_refbased.sample_name",
+                        "input_type": {
+                          "type": "optional",
+                          "optional_type": {
+                            "type": "primitive",
+                            "primitive_type": "String"
+                          }
+                        },
+                        "source": {
+                          "type": "record_lookup",
+                          "record_attribute": "sample_id"
+                        }
+                      },
+                      {
+                        "input_name": "assemble_refbased.plot_self_coverage.binning_summary_statistic",
+                        "input_type": {
+                          "type": "optional",
+                          "optional_type": {
+                            "type": "primitive",
+                            "primitive_type": "String"
+                          }
+                        },
+                        "source": {
+                          "type": "none"
```

Comment:
Definitely not a blocker, but out of curiosity, does case matter for type 'none'?

Reply:
I don't know - that's a good question. all lowercase is what we want, but maybe other cases would work too?